client/src/Components/relatedItems/relatedCarousel/relatedCard/cardComponents/actionButton.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Button({ handlesComparePopup }) {
+  return (
+    <div>
+      <button onClick={() => handlesComparePopup()}>Button</button>
```

Comment:
If 'handlesComparePopup' does not take in any arguments, it's not necessary to create a function that calls handlesComparePopup.

Instead, try 'onClick={handlesComparePopup}

Reply:
thanks for clarifying this!